samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/AssessmentEntityProducer.java

```diff
@@ -103,6 +112,7 @@ public class AssessmentEntityProducer implements EntityTransferrer, EntityProduc
     @Getter @Setter protected SiteService siteService;
     @Getter @Setter protected UserDirectoryService userDirectoryService;
     @Getter @Setter protected PublishedAssessmentFacadeQueriesAPI publishedAssessmentFacadeQueries;
+            @Setter protected LinkMigrationHelper linkMigrationHelper;
```

Comment:
Fix spacing? Not sure as it doesn't have getter but...

Reply:
I think it s a bit more readable with the extra indentation, but let's keep it standard.